### ISSUE TITLE
web.py doesn't initialize with AOO 4.1.2 on Win7

### ISSUE THREAD

**Issue by jmzambon** [hanya/MRI#9]
_OpenOffice 4.1.2 (i.e. python 2.7.6) on Windows 7._

MRI does not launch and complains with the following traceback: 
```
_"C:\Users\127998\AppData\Roaming\OpenOffice\4\user\uno_packages\cache\uno_packages
\sv59ol0.tmp_\MRI-1.3.3.oxt\pythonpath\mytools_Mri\web.py", line 104, in __init__
    super().__init__(cast, config)
TypeError: super() takes at least 1 argument (0 given)_
```
Modifying the line 104 with something like `super(DoxygenIDLRef, self).__init__(cast, config)
` works fine.
Thanks.

**Comment by hanya** [comment]
Wonder why you choose doxygen check box for AOO 4.1.2 reference. 
But fixed on rev 598c22c3406222eb605394e63eaba65e418e7599. 
Thank you for the report.

**Comment by jmzambon** [comment]
You're right, the doxygen option was checked by error, but as MRI didn't launch anymore, I wasn't able to uncheck it from the gui...
Thank for fixing it !
